lib/api/src/modules/refs.ts

```diff
@@ -168,10 +170,11 @@ export const init: ModuleFn = ({ store, provider, fullAPI }) => {
           `,
         } as Error;
       } else if (omitted || included) {
-        const credentials = !omitted ? 'include' : 'omit';
+        const credentials = included ? 'include' : 'omit';
 
-        const [stories, metadata] = await Promise.all([
-          handle(omitted || included),
+        const [stories, storiesWithAuth, metadata] = await Promise.all([
+          handle(omitted),
+          handle(included),
```

Comment:
```suggestion
          included ? handle(included) : handle(omitted),
```

No reason to complicate things, they are the "same" data, only one of them will work.

Reply:
Yea 1 of them works, this is how I know which one; and handle the data from the one that worked?

Reply:
Actually they both "work" 1 responds with `{ loginUrl: '' }` the other with `{ stories: {} }`.

Reply:
I had `included ? handle(included) : handle(omitted),` before and it keps sending me to the login-state because the `handle(omitted),` succeeded with `{ loginUrl: '' }`

Reply:
> Actually they both "work" 1 responds with { loginUrl: '' } the other with { stories: {} }.

If `included` doesn't error (with a CORS error) it should take priority over `omitted` no matter what. We should not even be looking at the result of `omitted` in such cases.

> I had included ? handle(included) : handle(omitted), before and it keps sending me to the login-state because the handle(omitted), succeeded with { loginUrl: '' }

I don't follow. If `included` worked then we would ignore `omitted` entirely.